Source/Scene/Cesium3DTileStyleEngine.js

```diff
@@ -113,6 +115,12 @@ define([
             return;
         }
 
+        //>>includeStart('debug', pragmas.debug);
+        if (!style.ready) {
```

Comment:
Then add a check for `style.ready` and quick return in `applyStyle` above just like it checks `tileset.ready`.